gulpfile.js

```diff
@@ -126,9 +119,19 @@ gulp.task('package', function (cb) {
   });
 });
 
-// Create bundled installer
+// Create stub installer that will then download all the requirements

```

Comment:
I think that you are not talking about the same thing :) 

1. clean should definitely run before task _package-simple_ and _package-bundle_
2. in task _packages_, it should run as first task, doesn't need to run after first installer is created